package.json

```diff
@@ -18,31 +18,29 @@
     "cozydb": "0.1.10",
     "graceful-fs": "3.0.8",
     "http-proxy": "1.12.0",
-    "jade": "1.11.0",
     "locale": "0.0.14",
     "lockedpath": "0.1.1",
     "node-polyglot": "0.3.0",
     "passport": "0.2.0",
     "passport-local": "1.0.0",
     "printit": "0.1.15",
+    "pug-runtime": "2.0.0",
```

Comment:
Seems that's the pug-runtime (formerly [jade-runtime](https://www.npmjs.com/package/jade-runtime), this renamming dance is madness) is still full compatible with jade@1. I didn't seen any regression during my tests.